gradle/build.go

```diff
@@ -98,7 +98,7 @@ func (b Build) Build(context libcnb.BuildContext) (libcnb.BuildResult, error) {
 		return libcnb.BuildResult{}, fmt.Errorf("unable to stat %s\n%w", command, err)
 	} else {
 		if err := os.Chmod(command, 0755); err != nil {
-			fmt.Printf("WARNING: unable to chmod %s:\n%s", command, err)
+			fmt.Printf("WARNING: unable to chmod %s:\n%s\n", command, err)
```

Comment:
```suggestion
			b.Logger.Body("WARNING: unable to chmod %s:\n%s", command, err)
```

Sorry, I missed this in the previous PR. This should be using the logger, which should take care of the EOL character.

Reply:
Done with `b.Logger.Bodyf`